### ISSUE TITLE
error on using nojump with async#cmd()

### ISSUE THREAD

**Issue by Konfekt** [mg979/tasks.vim#5]
The command

```vim
async#cmd('git push ' . <q-args>, 'quickfix', {'nojump': 1})
```

returns

```vim
Errore/i eseguendo function async#finish[4]..<SNR>415_cb_quickfix:
riga   57:
E730: uso di Lista come Stringa
E116: Argomenti non validi per la funzione: <SNR>415_echo
```

**Comment by mg979** [comment]
This command works ok:

     command! Test call async#cmd('git status ' . <q-args>, 'terminal', {'nojump': 1})

Both in vim and nvim. I don't know what's the problem you're having. I have to look at it a bit better.

**Comment by Konfekt** [comment]
I forgot removing an artifact. The proper command is:

```vim
call async#cmd('git push ', 'quickfix', {'nojump': 1})
```

giving

```
riga   57:
E730: uso di Lista come Stringa
E116: Argomenti non validi per la funzione: <SNR>424_echo
```

The fix is in #6 and occurs for `quickfix`, not `terminal` mode.
